direct/twindb.go

```diff
@@ -44,10 +45,14 @@ func (t *twinDB) Get(id uint32) (Twin, error) {
 	if err != nil {
 		return Twin{}, errors.Wrapf(err, "could net get twin with id %d", id)
 	}
-
+	relay := ""
+	if substrateTwin.Relay.HasValue {
+		relay = substrateTwin.Relay.AsValue
+	}
```

Comment:
that check is not needed, since the zero value for relay is empty string anyway, so it's safe to do 
```go
Twin{
  Relay: substrateTwin.Relay.AsValue
}
```

directly